en/docs/catalogs/crd.md

```diff
@@ -0,0 +1,4359 @@
+    </tbody>
+</table>
+<h3 id="dp.wso2.com/v1alpha1.Scope">Scope
+    <a class="headerlink" href="#dp.wso2.com%2fv1alpha1.Scope" title="Permanent link">¶</a>
+</h3>
+<p>
+<p>Scope is the Schema for the scopes API</p>
+</p>
+<table>
+    <thead>
+        <tr>
+            <th>Field</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+                <code>apiVersion</code></br>
+                string
+            </td>
+            <td>
+                <code>
+dp.wso2.com/v1alpha1
+</code>
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <code>kind</code></br>
+                string
+            </td>
+            <td><code>Scope</code></td>
+        </tr>
+        <tr>
+            <td>
+                <code>metadata</code></br>
+                <em>
+                    <a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#objectmeta-v1-meta">
+                        Kubernetes meta/v1.ObjectMeta
+                    </a>
+                </em>
+            </td>
+            <td>
+                Refer to the Kubernetes API documentation for the fields of the
+                <code>metadata</code> field.
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <code>spec</code></br>
+                <em>
+                    <a href="#dp.wso2.com/v1alpha1.ScopeSpec">
+                        ScopeSpec
+                    </a>
+                </em>
+            </td>
+            <td>
+                <br />
+                <br />
+                <table>
+                    <tr>
+                        <td>
+                            <code>names</code></br>
+                            <em>
+                                []string
+                            </em>
+                        </td>
+                        <td>
+                            <p>Name scope name</p>
+                        </td>
+                    </tr>
+                </table>
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <code>status</code></br>
+                <em>
+                    <a href="#dp.wso2.com/v1alpha1.ScopeStatus">
+                        ScopeStatus
+                    </a>
+                </em>
+            </td>
+            <td>
+            </td>
+        </tr>
+    </tbody>
+</table>
+<h3 id="dp.wso2.com/v1alpha1.TokenIssuer">TokenIssuer
+    <a class="headerlink" href="#dp.wso2.com%2fv1alpha1.TokenIssuer" title="Permanent link">¶</a>
+</h3>
+<p>
+<p>TokenIssuer is the Schema for the tokenIssuer API</p>
+</p>
+<table>
+    <thead>
+        <tr>
+            <th>Field</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+                <code>apiVersion</code></br>
+                string
+            </td>
+            <td>
+                <code>
+dp.wso2.com/v1alpha1
+</code>
```

Comment:
Shall we fix the indentation issue here under the description of apiVersion field? Let's check other places similar to this as well

<img width="855" alt="image" src="https://github.com/wso2/docs-apk/assets/30475839/22809ca2-5eac-4338-a2f6-6c8b5ea5cbe2">

Reply:
Fixed in PRs https://github.com/wso2/docs-apk/pull/583 and https://github.com/wso2/docs-apk/pull/584